src/style.css

```
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+body {
+  margin-top: 20px;
+}
+
+.big-type {
+  font-family: "Avenir", sans-serif;
```

Тут можеш унаслідувати
```suggestion
  font-family: inherit;
```